modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -580,7 +620,7 @@ std::vector<TensorInfo> ONNXCompiled::getTensorInfo(TensorPosition pos) {
 }
 
 cv::GMatDesc ONNXCompiled::outMeta(int idx) const {
-    if (is_dynamic) {
+    if (is_dynamic || is_postproc) {
```

Comment:
The assertion text below says `"Metadata must be specified if NN has dynamic inputs!"`, but this `if` is broader.

I mean, the assertion text should be extended with the new conditions.

Reply:
Comment extended.